### PR TITLE
Remove deprecated configs

### PR DESCRIPTION
**Pull request by ceyonur**
## Why this should be merged

Removes deprecated flags, which were deprecated for ~2 months

## How this works

Removes deprecated flags from keys and configs.

## How this was documented

https://github.com/ava-labs/avalanche-docs/pull/1450

### CODE REVIEW

**Comment by ceyonur** [comment]
We need a new ANR release to fix those tests.

**Review by StephenButtolph — APPROVED** [review]

File: config/config.go [hunk 1]
```diff
@@ -1509,12 +1485,3 @@ func providedFlags(v *viper.Viper) map[string]interface{} {
 	}
 	return customSettings
 }
-
-// getRenamedKey returns the new key if it is set, otherwise it returns the deprecated key.
-func getRenamedKey(v *viper.Viper, deprecatedKey string, newKey string) string {
```
Comment (joshua-kim) [inline config/config.go:1514]: This might be still useful to keep around if we deprecate keys in the future

File: scripts/install_anr.sh [hunk 2]
```diff
@@ -14,7 +14,7 @@ AVALANCHE_PATH=$(
 # https://github.com/ava-labs/avalanche-network-runner
 GOARCH=$(go env GOARCH)
 GOOS=$(go env GOOS)
-NETWORK_RUNNER_VERSION=1.3.9
```
Comment (joshua-kim) [inline scripts/install_anr.sh:17]: Why is this in the diff?

**Review by joshua-kim — COMMENTED** [review]
I think we will need a corresponding docs PR to remove the deprecated configs

**Review by dhrubabasu — APPROVED** [review]

File: scripts/install_anr.sh (hunk 2 above)
Comment (StephenButtolph) [reply scripts/install_anr.sh:17]: The older ANR versions used the deprecated `genesis` key.

File: config/config.go (hunk 1 above)
Comment (StephenButtolph) [reply config/config.go:1514]: I think it would fail linting as dead code

**Review by abi87 — APPROVED** [review]

**Pull request merged**